### DISCUSSION TITLE
How to fix indicator battery broken cause magisk module?

### DISCUSSION THREAD

**Discussion by Rheinner** [community/community#129834] [New to GitHub]
![Uploading Screenshot_20240626-173406.jpg…]()

Yesterday I installed the Magisk fast charging module, I've removed it but it still makes my battery indicator damaged and doesn't display the actual number of batteries, 0-90% normal charge but 90 - 100 jump very fast like 1% 1 minute

**Comment by ghostinhershell** [comment]
Hi @Rheinner,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this product or app. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
